docs-2.0/nebula-dashboard-en/1.what-is-dashboard-en.md

```diff
@@ -0,0 +1,47 @@
+# What is Nebula Dashboard Enterprise Edition
+
+Nebula Dashboard (Dashboard for short) is a visualization tool that monitors and manages the status of machines and services in Nebula Graph clusters. This topic introduces Dashboard Enterprise Edition. For details of Dashboard Community Edition, refer to [What is Nebula Dashboard Community Edition](../nebula-dashboard/1.what-is-dashboard.md).
+
+## Features
+
+- Create a Nebula Graph cluster of specified version, support import nodes in batch, add services with one click, and so on.
+
+- Support importing clusters, balancing data, scaling out or in, and so on through visualization interface.
+
+- Support manage clusters, and audit operation log of clusters within the last 14 days.
+
+- Support start, stop and restart services through visualization interface.
+
+- Support update the configuration of Storage services and Graph services in clusters quickly.
+
+- Monitors the information of all the services in clusters, including the IP address, version, and monitoring metrics (such as the number of queries, the latency of queries, the latency of heartbeats, and so on).
+
+- Monitors the status of all the machines in clusters, including CPU, memory, load, disk, and network.
+
+- Monitors the information of clusters, including the information of services, partitions, configurations, and long-term tasks.
+
+## Scenarios
+
+- You want a visualized operation and maintenance monitoring platform for large-scale clusters.
+
+- You want to monitor key metrics conveniently and quickly, and present multiple key information of the business to ensure the business operates normally.
+
+- You want to monitor clusters from multiple dimensions(such as the time, aggregate rules, and metrics).
+
+- after a failure occurs, you want to review it and confirm its occurrence time and unexpected phenomena.
+
+## Precautions
+
+- The monitoring data will be updated per 7 seconds by default.
+
+- The monitoring data will be retained for 14 days by default, that is, only the monitoring data within the last 14 days can be queried.
+
+- The version of Nebula Graph must be 2.0.1 or later.
+
+- Advised to use the latest version of Chrome to access Dashboard.
```

Comment:
use the complete form  
Advised --> it's advised to do  
but we use words suggest or recommend often in technical docs.